### ISSUE TITLE
IOS application

### ISSUE THREAD

**Issue by VetGeo** [VetGeo/Main#24]
It would be good if the software had an IOS application for our clinic's clients

**Comment by VetGeo** [comment]
This is in progress and will arrive soon

**Comment by jiaweizhang** [comment]
Are there plans to make an IOS app for the clinic employees? I know a vet who makes home visits and an IOS app would significantly simplify their work. Basically all (or at least most) functions also accessible from the phone.

**Comment by OliverDolan** [comment]
@jiaweizhang yes that is planned and will come in subsequent versions of the app
